woocommerce/woocommerce-ios
# [UI Tests] Moved the doc for UI tests to the UI Tests folder.

### Description
The documentation for WCiOS UI Tests was located together with the other docs for the repo. I think it makes sense to have the documentation in the place for which this documentation is written (it's done so for [WPiOS](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/WordPressUITests)) - since the person is more likely to find it there.

However, I do realize that there might be a reason to have the file in the original place too (cc @thehenrybyrd) - so I'd like to hear you thoughts too.

### Testing instructions
- Open [WooCommerceUITests](https://github.com/woocommerce/woocommerce-ios/tree/uitii-move-docs-to-dedicated-folder/WooCommerce/WooCommerceUITests) folder while on this branch, and make sure you see the `README.md` rendered automatically (UPD: and all links are functional).
- The PR does not change any app or tests logic, it just moves/renames a doc file to a different location. Having Buildkite 🟢  is good enough.
